src/main/webapp/i18n/en/guidedTourCodeEditor.json

```
@@ -0,0 +1,47 @@
+{
+    "tour": {
+        "programmingExercise": {
+            "fileBrowser": {
+                "headline": "File Browser",
+                "content": "In the file browser wou can view the project structure with all its folders and files. Additionally you can also create your own folders and files if necessary."
+            },
+            "exerciseFile": {
+                "headline": "Class file",
+                "content": "Click on the class file, so that its source code will be displayed in the code editor."
+            },
+            "codeEditor": {
+                "headline": "Code editor",
+                "content": "The source code is now displayed in the code editor. You can read through the TODO and try to solve it. Or alternatively you can enter any code for this test round. If you are done with the implementation, just click on the 'Next' button to proceed."
+            },
+            "saveChanges": {
+                "headline": "Save changes",
+                "content": "You can save your changes in the code editor by clicking on 'Save'."
+            },
+            "submitChanges": {
+                "headline": "Submit changes",
+                "content": "If you are content with your implementation, then you can submit your current code by clicking 'Submit'.",
+                "hint": "Here is a quick note: You can submit multiple times within the due date of the programming exercise, so you can always solve your tasks one by one and submit them or do everything in one commit based on your preference."
+            },
+            "buildLoading": {
+                "headline": "Build und tests",
+                "content": "After submitting your changes, Artemis will automatically trigger the build and the tests."
+            },
+            "buildStatus": {
+                "headline": "Build status",
+                "content": "Now you see your build status here and identify which tasks have passed the tests and are therefore successfully solved."
+            },
+            "testCases": {
+                "headline": "Feedback",
+                "content": "In this window you can view the feedback to your submission. Please try to solve all errors to pass this programming exercise successfully."
```

I think
```suggestion
                "content": "In this window you can view the feedback related to your submission. Please try to solve all errors to pass this programming exercise successfully."
```
or 
```suggestion
                "content": "In this window you can view the feedback based on your submission. Please try to solve all errors to pass this programming exercise successfully."
```
sound better